### DISCUSSION TITLE
Suggested revised tooltip for solar consumed gauge in energy dashboard

### DISCUSSION THREAD

**Discussion by wtadler** [home-assistant/frontend#9977] [Dashboard] [answered]
**Current look:**
![Screenshot of solar consumed gauge lovelace card](https://www.home-assistant.io/images/lovelace/energy/solar-consumed-gauge.png)
**Current tooltip text:**
` This card represents how much of the solar energy was used by your home and was not returned to the grid. If you frequently produce more than you consume, try to conserve this energy by installing a battery or buying an electric car to charge.`

First of all, it is not clear how to compare the tooltip with the graph. It takes a second to realize that “produce _more_ than you consume” counterintuitively refers to a situation where the displayed number is _low_. Secondly, I don’t think it’s correct to say that channeling energy into a battery or an electric car is “conserving” energy compared to returning it to the grid. Batteries are [not even necessarily advantageous](https://www.solarreviews.com/blog/do-you-save-more-money-with-net-metering-or-solar-batteries) for solar producers in areas (such as many US states) with good net metering policies, where the producer in effect treats the grid as a battery. Third, the text sort of makes excess production seem like a bad thing, which is hard to square with the green/red color choice in the grid neutrality gauge (though I think that should be changed anyway, see #9976). So my suggested text is:
`This card indicates how much of the solar energy you produced was used by your home instead of being returned to the grid. If this number is typically very low, indicating excess solar production, you might want to consider charging a home battery or electric car from your solar panels at times of high solar production.`

**Comment by wowulavega** [comment]
Hi, 
to me, it is not really clear, how to calculate the sensors out of Solar production , grid support, and energy consumption.
Is there any link, you could share, i’ve been looking now for a week but can’t find any instructions and my effort so fare leads to an error that a sensor generates a negative value…. which is normal, when i generate more than I use in certain times.
Thanks for you’re assistance .

**Comment by wtadler** [comment, marked as answer]
Addressed by #10052
